equinix/resource_metal_organization_member.go

```diff
@@ -0,0 +1,270 @@
+}
+
+func resourceMetalOrganizationMember() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMetalOrganizationMemberCreate,
+		Read:   resourceMetalOrganizationMemberRead,
+		Delete: resourceMetalOrganizationMemberDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				parts := strings.Split(d.Id(), ":")
+				invitee := parts[0]
+				orgID := parts[1]
+				d.SetId(d.Id())
+				d.Set("invitee", invitee)
+				d.Set("organization_id", orgID)
+				if err := resourceMetalOrganizationMemberRead(d, meta); err != nil {
+					return nil, err
+				}
+				if d.Id() == "" {
+					return nil, fmt.Errorf("Member %s does not exist in organization %s.", invitee, orgID)
+				}
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+
+		Schema: map[string]*schema.Schema{
+			"invitee": {
+				Type:         schema.TypeString,
+				Description:  "The email address of the user to invite",
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+			"invited_by": {
+				Type:        schema.TypeString,
+				Description: "The email address of the user invited by",
+				Computed:    true,
+			},
+			"organization_id": {
+				Type:         schema.TypeString,
+				Description:  "The organization to invite the user to",
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+			"projects_ids": {
+				Type:        schema.TypeSet,
+				Description: "Project IDs the member has access to within the organization. If the member is an 'admin', the projects list should be empty.",
+				Required:    true,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+				// TODO: Update should be supported. packngo.InvitationService does not offer an Update	method.
+				ForceNew: true,
+			},
+			"nonce": {
+				Type:        schema.TypeString,
+				Description: "The nonce for the invitation (only known in the invitation stage)",
+				Computed:    true,
+			},
+			"created": {
+				Type:        schema.TypeString,
+				Description: "When the invitation was created (only known in the invitation stage)",
+				Computed:    true,
+			},
+			"updated": {
+				Type:        schema.TypeString,
+				Description: "When the invitation was updated (only known in the invitation stage)",
+				Computed:    true,
+			},
+			"roles": {
+				Type:        schema.TypeSet,
+				Description: "Organization roles",
+				Required:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				// TODO: Update should be supported. packngo.InvitationService does not offer an Update	method.
+				ForceNew: true,
+			},
+			"state": {
+				Type:        schema.TypeString,
+				Description: "The state of the membership ('invited' when an invitation is open, 'active' when the user is an organization member)",
+				Computed:    true,
+			},
+		},
```

Comment:
nit: the `message` field was not included. I'm honestly not sure if this even appears in the emailed invite or anywhere else.